docs/developer-docs/latest/update-migration-guides/migration-guides/v4/code/backend/routes.md

```diff
@@ -0,0 +1,88 @@
+
+Due to the differences between routes implementation in Strapi v3 and v4, it's recommended to create a new router file, then optionally bring existing v3 customizations into the new file and adapt them when necessary.
+
+A new router file can be created:
+
+- manually, starting at step 1 of the following procedure,
+- or automatically, using the [`strapi generate` interactive CLI](/developer-docs/latest/developer-resources/cli/CLI.md#strapi-generate) then jumping to step 2 of the following procedure.
+<!-- ? I noticed the strapi generate CLI generates an empty router and not a call to createCoreRouter, is it intended ? -->
+
+To create a v4 router file:
+
+1. Create a `api/<api-name>/routes/<router-name>.js` file inside the `./src` folder (see [project structure](/developer-docs/latest/setup-deployment-guides/file-structure.md)).
+
+2. Copy and paste the following code at the top of the `./src/api/<api-name>/routes/<router-name>.js` file. The code imports the `createCoreRouter` factory function from the factories included with the core of Strapi:
+
+    ```js
+    const { createCoreRouter } = require('@strapi/strapi').factories;
+    ```
+
+3. Copy and paste the following code, replacing `api-name` and `content-type-name` with appropriate names. The code exports the result of a call to the `createCoreRouter` factory function, passing the unique identifier of the content-type (e.g. `api::api-name.content-type-name`) as an argument:
+
+    ```js
+    module.exports = createCoreRouter('api::api-name.content-type-name')
+    ```
+
+4. (_optional_) To configure the router, pass a second argument to the `createCoreRouter` factory function. This argument can be either an object or a function returning an object. The object contains methods, which can either be entirely new actions or replace or extend existing actions of core API routes (see [routes implementation documentation](/developer-docs/latest/development/backend-customization/routes.md#implementation)).
+
+::: details Example of a v4 router without customization
+
+  ```jsx
+  // path: ./src/api/<content-type-name>/routes/<router-name>.js
+
+  const { createCoreRouter } = require('@strapi/strapi').factories;
+
+  module.exports = createCoreRouter('api::api-name.content-type-name');
+  ```
+
+:::
+
+::: details Example of a v4 router with customization
+
+  ```jsx
+  // path: ./src/api/<content-type-name>/routes/<router-name>.js
+
+  const { createCoreRouter } = require('@strapi/strapi').factories;
+
+  module.exports = createCoreRouter('api::foo.foo', {
+    // ...
+    config: {
+      find: {
+        policies: ['admin::isAuthenticatedAdmin'],
+        // here you can also customize auth & middlewares
+      },
+    },
+    only: ['find', 'findOne'],
+  });
+
+  ```
+
+:::
+
+:::tip Customization tips
```

Comment:
Doesn't apply to routes only controllers and services. Looks like a bad copy paste :)

Reply:
Indeed! Forgot to push [the latest commit](https://github.com/strapi/documentation/pull/720/commits/203eac28472c9939ba9e0cadb9fac182efedbd10) 🤦‍♂️